addons/l10n_au/data/template/account.tax-au.csv

```diff
@@ -1,110 +1,110 @@
-"","","","","","","","","","","","","tax","invoice","au_21330","+ONLY||+GST from General Ledger"
-"","","","","","","","","","","","","base","refund","",""
-"","","","","","","","","","","","","tax","refund","au_21330","-ONLY||-GST from General Ledger"
-"au_tax_purchase_gst_only_service_tpar","100% EX TPAR","8","GST Only on Imports","GST Only on Imports","purchase","service","division","100.0","True","tax_group_gst_100000000","","base","invoice","",""
-"","","","","","","","","","","","","tax","invoice","au_21330","+ONLY||+GST from General Ledger||Service"
-"","","","","","","","","","","","","base","refund","",""
-"","","","","","","","","","","","","tax","refund","au_21330","-ONLY||-GST from General Ledger||Service"
-"au_tax_purchase_gst_only_service_tpar_no_abn","100% EX TPAR N ABN","8","GST Only on Imports","GST Only on Imports","purchase","service","group","100.0","","tax_group_gst_10","au_tax_purchase_gst_only_service_tpar,au_tax_witheld","","","",""
-"au_tax_purchase_adj_service","10% Adj","9","Tax Adjustments (Purchases)","Tax Adjustments (Purchases)","purchase","","percent","10.0","False","tax_group_gst_10","","base","invoice","","+G18"
-"","","","","","","","","","","","","tax","invoice","au_21330","+G18||+GST from General Ledger"
-"","","","","","","","","","","","","base","refund","","-G18"
-"","","","","","","","","","","","","tax","refund","au_21330","-G18||-GST from General Ledger"
-"au_tax_purchase_adj_service_tpar","10% Adj TPAR","9","Tax Adjustments (Purchases)","Tax Adjustments (Purchases)","purchase","service","percent","10.0","False","tax_group_gst_10","","base","invoice","","+G18"
-"","","","","","","","","","","","","tax","invoice","au_21330","+G18||+GST from General Ledger||Service"
-"","","","","","","","","","","","","base","refund","","-G18"
-"","","","","","","","","","","","","tax","refund","au_21330","-G18||-GST from General Ledger||Service"
-"au_tax_purchase_adj_service_tpar_no_abn","10% Adj TPAR N ABN","9","Tax Adjustments (Purchases)","Tax Adjustments (Purchases)","purchase","service","group","100.0","","tax_group_gst_10","au_tax_purchase_adj_service_tpar,au_tax_witheld","","","",""
+"id","name","sequence","description","invoice_label","type_tax_use","tax_scope","amount_type","amount","price_include","tax_group_id","children_tax_ids","repartition_line_ids/repartition_type","repartition_line_ids/document_type","repartition_line_ids/account_id","repartition_line_ids/tag_ids","active"
+"au_tax_witheld","47% WH","1000","Tax Withheld for Partners without ABN","Tax Withheld for Partners without ABN","purchase","service","percent","-47.0","False","tax_group_withheld","","base","invoice","","","True"
+"","","","","","","","","","","","","tax","invoice","au_21380","l10n_au.tax_withheld_tag","True"
+"","","","","","","","","","","","","base","refund","","","True"
+"","","","","","","","","","","","","tax","refund","au_21380","l10n_au.tax_withheld_tag","True"

```

Comment:
close but the field should only be set on the tax itself and not on the repartition lines :)